docs/integrations/catalog/prefect-spark-on-k8s-operator.yaml

```
@@ -0,0 +1,6 @@
+collectionName: prefect-spark-on-k8s-operator
+author: Manoj Babu Katragadda
+authorUrl: https://github.com/tardunge
+documentation: https://tardunge.github.io/prefect-spark-on-k8s-operator/
+iconUrl: /img/collections/spark-on-kubernetes.png
+tag: SparkOnK8sOperator
```

Can you update the `tag` so that a human-friendly name displays?
```suggestion
tag: Spark on Kubernetes
```